### PR TITLE
insert vertex if not exists

### PR DESCRIPTION
**Pull request by nevermore3**
wait storage & graph..

### CODE REVIEW

File: src/common/interface/storage.thrift [hunk 1]
```diff
@@ -406,8 +406,9 @@ struct AddVerticesRequest {
     //   in the NewVertex.NewTag.props
     3: map<common.TagID, list<binary>>
         (cpp.template = "std::unordered_map")   prop_names,
+    4: bool                                     if_not_exists,
     // If true, it equals an (up)sert operation.
-    4: bool                                     overwritable = true,
+    5: bool                                     overwritable = true,
```
Comment (panda-sheep) [inline src/common/interface/storage.thrift:411]: What is the difference between if_not_exists and overwritable?

Comment (nevermore3) [reply src/common/interface/storage.thrift:411]: when the (vertexID, tagID) already exists,   overwriteable do update and if_not_exists do nothing

Comment (panda-sheep) [reply src/common/interface/storage.thrift:411]: In my opinion, overwriteable is not currently used,
The effect of overwriteable should be the same as if_not_exists. WDYT?

Comment (nevermore3) [reply src/common/interface/storage.thrift:411]: Agree, maybe we can delete overwriteable, if_not_exist is easier to understand

Comment (panda-sheep) [reply src/common/interface/storage.thrift:411]: Another question
Here is to write records to the table, why only process the `tag `table, not the `edge `table?

As follows:
image

Comment (nevermore3) [reply src/common/interface/storage.thrift:411]: do it later

**Review by laura-ding — APPROVED** [review]

**Review by panda-sheep — APPROVED** [review]

**Review by yixinglu — APPROVED** [review]

**Pull request merged**